app/views/service_providers/show.html.erb

```diff
@@ -58,27 +58,48 @@
   <% end %>
 <% end %>
 
-    <% if service_provider.identity_protocol == 'saml' %>
-    <h2><label for="acs_url">Assertion Consumer Service URL:</label></h2>
-<p class="font-mono-xs margin-top-0" name="acs_url"><%= service_provider.acs_url %></p>
+<% if service_provider.identity_protocol == 'saml' %>
+  <h2><label for="acs_url">Assertion Consumer Service URL:</label></h2>
+  <p class="font-mono-xs margin-top-0" name="acs_url"><%= service_provider.acs_url %></p>
 
-<h2><label for="assertion_consumer_logout_service_url">Assertion Consumer Logout Service URL:</label></h2>
-<p class="font-mono-xs margin-top-0" name="assertion_consumer_logout_service_url"><%= service_provider.assertion_consumer_logout_service_url %></p>
+  <h2><label for="assertion_consumer_logout_service_url">Assertion Consumer Logout Service URL:</label></h2>
+  <p class="font-mono-xs margin-top-0" name="assertion_consumer_logout_service_url"><%= service_provider.assertion_consumer_logout_service_url %></p>
 
-<h2><label for="assertion_consumer_logout_service_url">SP Initiated Login URL:</label></h2>
-<p class="font-mono-xs margin-top-0" name="assertion_consumer_logout_service_url"><%= service_provider.sp_initiated_login_url %></p>
+  <h2><label for="assertion_consumer_logout_service_url">SP Initiated Login URL:</label></h2>
+  <p class="font-mono-xs margin-top-0" name="assertion_consumer_logout_service_url"><%= service_provider.sp_initiated_login_url %></p>
 
-<h2><label for="block_encryption">SAML Assertion Encryption:</label></h2>
-<p class="font-mono-xs margin-top-0" name="block_encryption"><%= service_provider.block_encryption %></p>
-    <% end %>
+  <h2><label for="block_encryption">SAML Assertion Encryption:</label></h2>
+  <p class="font-mono-xs margin-top-0" name="block_encryption"><%= service_provider.block_encryption %></p>
+<% end %>
+
+<h2>
+  <label for="saml_client_cert">Public certificate:</label>
```

Comment:
It existed previously, though the `name` on the new `<details>` had caught my attention: Does the `for` attribute actually do anything for us? This doesn't seem like an appropriate use of a label, since it's not associated with an input.

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable

I could maybe see it as a case for [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) or [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute).

Reply:
Looks like they came from #366 from AxE matchers

Reply:
Ok no that was not right... looks like those have just been there a long time.... it's probably time for them to go, especially since this page isn't a form?

Reply:
I wonder if they were copied over from the edit form?